cmd/antrea-agent/agent.go

```diff
@@ -343,6 +343,8 @@ func run(o *Options) error {
 			mcNamespace,
 			o.config.Multicluster.EnableStretchedNetworkPolicy,
 			o.config.Multicluster.EnablePodToPodConnectivity,
+			o.config.Multicluster.WireGuard,
```

Comment:
Should we pass the whole `o.config.Multicluster` to NewMCRouteController() rather than four separate arguments, to reduce number of arguments.